app/index.html

```diff
@@ -17,15 +17,7 @@
   <body>
 
     <div class="container wrap">
-      <div class="row">
```

Comment:
Nice job creating a directive for the header.  Cleans up the index.html significantly